moduleroot/.github/workflows/release.yml.erb

```diff
@@ -0,0 +1,57 @@
+<% common = config_for('common') -%>
+name: "Publish module"
+
+on:
+  workflow_dispatch:
+    inputs:
+      forge_api_key:
+        description: "Forge API token to upload module to the Puppet Forge"
```

Comment:
can we set this as github secret instead?

Reply:
Initial approach was to have it as a Github secret, but having it as an input might be easier for the future.
I'm open to suggestions here.

Reply:
If you want to preserve it as input for testing/one-off releases, we could just do both and use `${FORGE_TOKEN:-${{ env.FORGE_TOKEN }}}` (see "Use Default Values" on https://linux.die.net/man/1/bash) at the call site

Reply:
Sounds like a good idea, I'll add that in

Reply:
Having issues masking the input token, so for initial workflow we'll use only the repository secret